SalesforceCommerceCloud/pwa-kit
# [Shopper Experience] `ImageTile` component

# Description



The PR adds the equivalent SFRA Page designer `ImageTile` component to the PWA.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added simple ImageTile component and test.

# How to Test-Drive This PR
1. Start the Retail React template
```
npm run start --prefix packages/template-retail-react-app/
```
2. Navigate to url http://localhost:3000/experience
3. Verify the component works as expected.
4. Confirm the same SFRA functionality is implemented and tests pass.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
